cmd/podman/common/netflags.go

```diff
@@ -53,6 +53,13 @@ func DefineNetFlags(cmd *cobra.Command) {
 	)
 	_ = cmd.RegisterFlagCompletionFunc(ipFlagName, completion.AutocompleteNone)
 
+	ip6FlagName := "ip6"
+	netFlags.String(
+		ip6FlagName, "",
+		"Specify a static IPv6 address for the container",
+	)
+	_ = cmd.RegisterFlagCompletionFunc(ip6FlagName, completion.AutocompleteNone)
+
```

Comment:
Ugh, I hate the ip6 name, but it's compliant with Docker.  I'd much prefer ipv6, but that's probably just my OCD kicking in.

Reply:
well we have to live with it